src/endpoints/private/system-country-codes/SystemCountryCodes.types.ts

```diff
@@ -1,6 +1,13 @@
+import {type CarrierId} from '@myparcel/constants';
+
 export interface SystemCountryCode {
   label: string;
   region: string;
 }
 
 export type SystemCountryCodesPerCountry = Record<string, SystemCountryCode>[];
+
+export type GetSystemCountryCodesParams = {
+  carrier_id?: CarrierId;
+  unfiltered?: boolean;
```

Comment:
Als `unfiltered` niet wordt meegegeven wanneer we de API call maken, wordt er dan in de API een fallback value voor gebruikt?

Zo ja, kan er dan iets kapot gaan in de front-end door niet gewenste data?

Reply:
geen idee ik zag alleen dat dit in de admin wel meegegeven word en in de backoffice niet. Wat er in de api gebeurt als je het niet meegeeft weet ik niet